### PR TITLE
2022 update

### PR DESCRIPTION
**Pull request by wvictor14**
Hi @jepa! 

Thank you for making an amazing UBC thesisdown template! I am happy to report that I've successfully used it to write my thesis (just defended!). 

In this pull request, I plan to merge some of the formatting changes that I had to make when UBC-FOGS reviewed my thesis. The goal is for this pull request to fully satisfy the 2022 formatting requirements. I also added a few examples, clarifications, and suggestions to make the template more user-friendly and accessible.

I am going to add to the list of changes here for easy tracking:

- [x]  Change title of table of contents from "Contents" to "Table of Contents" 96cef6f
- [x]  Add ["Additional Supervisory Committee Members"](https://www.grad.ubc.ca/current-students/dissertation-thesis-preparation/preliminary-pages) section to committee form 70de7a6
- [x] in TOC: appendices should be subsectioned (Under "Appendices") and unnumbered, with full titles 57f15c6
eg:

Appendices .........
(indent) Appendix A: Supplementary information for Chapter 2.......
(indent) Appendix B: Supplementary information for Chapter 3.......
etc .........

- [x] Glossary needs horizontal lines removed. Reads in a`glossary.csv` directly in rmd. It's easier this way, because in the rmd, the acronyms and terms are not on the same line, which is difficult for large glossary. 5216546 9f84eec
- [x] Remove the text "Chapter (#)" part of chapter titles: e5594e7
Should be "Introduction" not "Chapter One: Introduction" 96cef6f
---------------------------------------------------------------------------------

\+ Some README typos, errors, and clarifications. 562080b 562080b

---------------------------------------------------------------------------------

Additional edits, that are technically not necessary, but very useful:
- An example on how to use in-text references for figure captions. Useful if some captions use citations. 9eb589c
- An example on how to use tables, and some useful KableExtra features 8404fdb
- An example on how to use/add citations, and how to switch citation styles 4b6c466
- Added the Vancouver reference style which is more appropriate for most programs in the Sciences (MEDG, GSAT, IOP, BIOF) 3eb854b
- Link citations, a nice feature imo
- Added `readr` to install instructions, since I suggest using .csv to load tables 47ffce1
- Added a `NEWS.md` file to document major updates, such as this one 8f18b73
- Noted that some LaTex knowledge is required to use the template, such as in-text references. I believe most can get away with learning ~1% of LaTex which is not that difficult.  47ffce1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,25 +17,26 @@ From here on, the instructions below are the same regardless of the method you c
...
 # after restarting RStudio, confirm that you have LaTeX with 
 tinytex:::is_tinytex()
 ```
 
-If you are getting an error message “destination /usr/local/bin not writable” you can see the following [`tidytex`](https://github.com/yihui/tinytex/issues/24) issue.
+If you are getting an error message “destination /usr/local/bin not writable” you can see the following [`tinytex`](https://github.com/yihui/tinytex/issues/24) issue.
 
 
-Note that this template was created using `R version 3.5.2 (2018-12-20) -- "Eggshell Igloo"`  and tested in `R version 4.0.4 (2021-02-15) -- "Lost Library Book"` 
+Note that this template was created and tested using `R version 4.1.1 (2021-08-10) -- "Eggshell Igloo"`.
```
Comment (jepa) [inline README.md:33]: Please do not erased the versions but add them.

File: main_script.Rmd [hunk 2]
```diff
@@ -16,8 +16,11 @@ header-includes:
    - \floatplacement{figure}{H} 
    - \usepackage{wrapfig}
    - \usepackage{pdfpages}
+   - \usepackage{caption} # allows setting caption width for tables
 bibliography: ./References/reference_list.bib
-csl: ./References/reference_format.csl
+csl: ./References/reference_format-Vancouver.csl
```
Comment (jepa) [inline main_script.Rmd:21]: Should change name to "reference_format_vancouver.csl" for consistency

**Review by jepa — CHANGES_REQUESTED** [review]
These all look good @wvictor14 , thank you for contributing. Can you change the couple of things I highlighted here?

**Comment by wvictor14** [comment]
Thanks @jepa I'm awaiting UBC to get back to me on a review of my thesis format. I will make any additional changes they suggest to this branch and reopen the PR if needed. 

Victor

**Pull request merged**